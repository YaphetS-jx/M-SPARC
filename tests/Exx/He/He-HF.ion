ATOM_TYPE: He
PSEUDO_POT: He.pot
N_TYPE_ATOM: 1
COORD:
10 10 10
