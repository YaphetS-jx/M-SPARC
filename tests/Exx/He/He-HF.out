***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 05-Jan-2021  Start time: 16:34:40                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 20.000000 20.000000 
FD_GRID: 67 67 67
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.000999993
CHEB_DEGREE: 26
NSTATES: 2
EXCHANGE_CORRELATION: HF
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 1
PRINT_DENSITY: 0
MAXIT_FOCK: 20
TOL_FOCK: 1.00E-06
HYB_MIXING: 1.00
EXX_METHOD: FOURIER_SPACE
OUTPUT_FILE: ../tests/Exx/He/He-HF.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.298507 (Bohr)
Output printed to                  :  ../tests/Exx/He/He-HF.out
Total number of atom types         :  1
Total number of atoms              :  1
Total number of electrons          :  2
Atom type 1  (valence electrons)   :  He 2
Pseudopotential                    :  He.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.87 6.87 6.87
Number of atoms of type 1          :  1
Estimated total memory usage       :  1.17 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7898729035E+00        1.109E-01        6.364
2            -2.7899721013E+00        5.515E-02        6.240
3            -2.7899727337E+00        4.439E-02        6.299
4            -2.7899727417E+00        4.032E-02        5.258
5            -2.7899727435E+00        4.174E-02        3.820
6            -2.7899727440E+00        4.114E-02        4.135
7            -2.7899727442E+00        3.798E-02        3.386
8            -2.7899727443E+00        3.344E-02        3.030
9            -2.7899727443E+00        1.518E-02        3.055
10           -2.7899727445E+00        1.351E-02        3.569
11           -2.7899727446E+00        1.166E-02        2.698
12           -2.7899727442E+00        1.093E-02        2.808
13           -2.7899727448E+00        1.112E-02        2.797
14           -2.7899727440E+00        9.475E-03        2.410
15           -2.7899727448E+00        5.957E-03        2.783
16           -2.7899727451E+00        3.543E-03        2.346
17           -2.7899727449E+00        4.816E-03        2.421
18           -2.7899727447E+00        3.634E-03        2.087
19           -2.7899727488E+00        4.049E-03        2.388
20           -2.7899727446E+00        2.195E-03        1.976
21           -2.7899727444E+00        4.333E-03        2.395
22           -2.7899727438E+00        6.055E-03        2.045
23           -2.7899727433E+00        3.141E-03        2.371
24           -2.7899727452E+00        4.630E-04        1.989
25           -2.7899727439E+00        2.407E-04        2.161
26           -2.7899727401E+00        4.965E-04        2.018
27           -2.7899727435E+00        1.249E-04        1.686
28           -2.7899727442E+00        4.254E-04        2.049
29           -2.7899727431E+00        1.319E-04        2.023
30           -2.7899727497E+00        2.737E-05        1.706
31           -2.7899727498E+00        3.943E-05        1.772
32           -2.7899727491E+00        1.558E-05        1.771
33           -2.7899727496E+00        8.662E-06        1.691
34           -2.7899727497E+00        9.156E-07        1.713
Total number of SCF: 34    
No.1 Exx outer loop:
1            -3.7216680253E+00        1.847E-02        25.478
2            -3.7216728536E+00        1.188E-02        9.618
3            -3.7216755534E+00        7.298E-04        8.725
4            -3.7216756000E+00        1.578E-05        9.066
5            -3.7216756009E+00        3.464E-05        8.732
6            -3.7216755999E+00        5.723E-06        7.042
7            -3.7216756035E+00        2.702E-06        6.467
8            -3.7216756016E+00        2.638E-06        5.686
9            -3.7216755996E+00        4.346E-07        5.714
Total number of SCF: 9     
 Eex = 0.95692122
 Etot = -2.76475437
No.2 Exx outer loop:
1            -3.7276152220E+00        6.183E-03        23.509
2            -3.7276160881E+00        3.878E-03        9.102
3            -3.7276165713E+00        7.568E-06        8.473
4            -3.7276165709E+00        1.610E-06        8.503
5            -3.7276165735E+00        9.363E-07        6.464
Total number of SCF: 5     
 Eex = 0.96090495
 Etot = -2.76671163
No.3 Exx outer loop:
1            -3.7297040548E+00        2.186E-03        23.431
2            -3.7297041928E+00        1.349E-03        8.805
3            -3.7297042689E+00        4.127E-06        8.303
4            -3.7297042682E+00        1.136E-06        8.632
5            -3.7297042660E+00        7.392E-07        6.104
Total number of SCF: 5     
 Eex = 0.96232473
 Etot = -2.76737954
No.4 Exx outer loop:
1            -3.7304621238E+00        7.846E-04        23.374
2            -3.7304621443E+00        4.798E-04        8.501
3            -3.7304621556E+00        3.397E-07        8.170
Total number of SCF: 3     
 Eex = 0.96284113
 Etot = -2.76762103
No.5 Exx outer loop:
1            -3.7307377627E+00        2.829E-04        23.490
2            -3.7307377659E+00        1.722E-04        8.648
3            -3.7307377672E+00        7.065E-07        7.506
Total number of SCF: 3     
 Eex = 0.96302894
 Etot = -2.76770882
No.6 Exx outer loop:
1            -3.7308378670E+00        1.020E-04        23.411
2            -3.7308378681E+00        6.195E-05        8.191
3            -3.7308378683E+00        7.236E-07        6.842
Total number of SCF: 3     
 Eex = 0.96309712
 Etot = -2.76774075
No.7 Exx outer loop:
1            -3.7308741243E+00        3.663E-05        23.413
2            -3.7308741250E+00        2.224E-05        7.846
3            -3.7308741247E+00        2.857E-07        6.423
Total number of SCF: 3     
 Eex = 0.96312179
 Etot = -2.76775233
No.8 Exx outer loop:
1            -3.7308872243E+00        1.319E-05        23.389
2            -3.7308872252E+00        8.036E-06        7.166
3            -3.7308872248E+00        2.772E-07        6.107
Total number of SCF: 3     
 Eex = 0.96313071
 Etot = -2.76775652
No.9 Exx outer loop:
1            -3.7308919520E+00        4.711E-06        23.205
2            -3.7308919526E+00        2.882E-06        6.538
3            -3.7308919531E+00        8.626E-08        6.129
Total number of SCF: 3     
 Eex = 0.96313393
 Etot = -2.76775802
No.10 Exx outer loop:
1            -3.7308936699E+00        1.653E-06        23.412
2            -3.7308936708E+00        1.030E-06        6.139
3            -3.7308936720E+00        3.570E-08        5.741
Total number of SCF: 3     
 Eex = 0.96313510
 Etot = -2.76775857
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.7677585729E+00 (Ha/atom)
Total free energy                  : -2.7677585729E+00 (Ha)
Band structure energy              : -1.8046228545E+00 (Ha)
Exchange correlation energy        : -0.0000000000E+00 (Ha)
Self and correction energy         : -3.4566424553E+00 (Ha)
Entropy*kb*T                       : -3.3300186621E-10 (Ha)
Fermi level                        : -8.8372864459E-01 (Ha)
Average force                      :  0.0000000000E+00 (Ha/Bohr)
Maximum force                      :  0.0000000000E+00 (Ha/Bohr)
Time for force calculation         :  1.625 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  651.682 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
